MIP/mip-15/README.md

```diff
@@ -1,6 +1,7 @@
 # MIP-15: MG (Movement Gloss)
 - **Description**: Introduces the Movement Gloss (MG) document type and establishes a process for introducing new glossary terms.
```

Comment:
Add specification for adding entries into the `GLOSSARY.md` into the MIP.

Reply:
I added a specification further down in the document.